salt/modules/zabbix.py

```diff
@@ -1398,14 +1398,14 @@ def host_update(hostid, **connection_args):
         return ret
 
 
-def host_inventory_get(hostids, **connection_args):
+def host_inventory_get(hostid, **connection_args):
```

Comment:
**question** Does this affect existing zabbix users? If so we'll want to deprecate the hostids (it would still be in connection_args)

Reply:
whoops, see that was reverted :joy: